src/pygats/recog.py

```diff
@@ -0,0 +1,360 @@
+                lines[i] += ' ' + splitted2[11]
+
+
+def combine_lines(lines):
+    """Function translate lines from Tesseract output format into
+    result tuple
+
+    Args:
+        lines (List): Returns result containing box boundaries, confidences,
+            and other information.
+
+    Returns:
+        list: combined tuples
+
+    Notes:
+        There is magic number 5 to understand if words on the same line.
+        It should be reworked in future.
+
+    Todo:
+        * This function should be reworked in future with
+          combine_words_in_lines. Need one function to combine words in
+          sentences.
+    """
+    result = []
+    for i in range(1, len(lines)-1):
+        splitted = lines[i].split('\t')
+        if len(splitted) != 12:
+            return result
+        x = int(splitted[6])
+        y = int(splitted[7])
+        w = int(splitted[8])
+        h = int(splitted[9])
+        text = splitted[11]
+        for j in range(i+1, len(lines)-1):
+            splitted2 = lines[j].split('\t')
+            if abs(y - int(splitted2[7])) < 5:
+                if len(splitted2[11].strip()) > 0:
+                    w += int(splitted[8])
+                    text += ' ' + splitted2[11]
+        result.append((x, y, w, h, text))
+    return result
+
+
+def find_text(img, txt, skip=0, inv = False, one_word = False):
+    """Function finds text in image with Tesseract
+
+    Args:
+        img (PIL.Image): image where text will be recognized
+        text (pygats.recog.SearchedText): text which fill be searched
+        skip (int): amount of skipped finding
+
+    Returns:
+        (x,y,w,h,text):
+            x (int), y (int): coordinates of top-left point of rectangle where
+               text resides
+            w (int), h (int): width and height of rectangle where text resides
+            text (string): full text which resides in rectangle
+
+    """
+    cv_image = np.array(img)
+    img_gray = cv.cvtColor(cv_image, cv.COLOR_BGR2GRAY)
+    if inv:
+        thresh = cv.adaptiveThreshold(img_gray, 255,
+                                      cv.ADAPTIVE_THRESH_MEAN_C, \
+                                      cv.THRESH_BINARY_INV, 11, 2)
+    else:
+        thresh = cv.adaptiveThreshold(img_gray, 255,
+                                      cv.ADAPTIVE_THRESH_MEAN_C, \
+                                      cv.THRESH_BINARY, 11, 2)
+    recognized = pytesseract.image_to_data(thresh, txt.lang).split('\n')
+    if one_word is False:
+        combine_words_in_lines(recognized)
+    ret_tuple = (-1, -1, -1, -1, False)
+    for line in recognized[1:]:
+        splitted = line.split('\t')
+        if len(splitted) == 12:
+            if splitted[11].find(txt.text) != -1:
+                print("Найден текст " + splitted[11])
+                ret_tuple = (int(splitted[6]), int(splitted[7]), int(
+                    splitted[8]), int(splitted[9]), True)
```

Comment:
must either put the lower arguments under the arguments of the upper line, or move the arguments of the upper line to the next line